.viperlightignore

```diff
@@ -26,6 +26,8 @@ source/patterns/@aws-solutions-constructs/aws-lambda-dynamodb/test/lambda-dynamo
 source/patterns/@aws-solutions-constructs/aws-lambda-sqs-lambda/test/lambda-sqs-lambda.test.ts:66
 source/patterns/@aws-solutions-constructs/aws-lambda-step-function/test/lambda-step-function.test.ts:129
 source/patterns/@aws-solutions-constructs/aws-lambda-stepfunctions/test/lambda-stepfunctions.test.ts:129
```

Comment:
I think these lines were replaced by the 130 lines, correct? Can these now be removed?